contributors/chairs-and-techleads/leadership-changes.md

```diff
@@ -0,0 +1,40 @@
+# Leadership Changes
+
+This document covers steps needed to propose changes to SIG/WG/UG leadership.
+
+- [ ] Discuss the proposed changes with the current leadership.
+
+- [ ] Send an email to the SIG/WG/UG mailing list and cc the [kubernetes-dev]
+mailing list. At a minimum, the email should contain:
+  - [ ] intent to step down as the current lead
+  - [ ] if nominating another lead
+    - [ ] 1-2 lines about why they are being nominated
+    - [ ] contacts to privately reach out to for questions (current leads)
+          or concerns (current leads + [steering-private]) about the nomination
+  - [ ] if this was discussed in a meeting, link to meeting notes
+  - [ ] lazy consensus deadline of at least one week
+
+- [ ] If nominating another lead, ensure that they
+  - [ ] are a Kubernetes GitHub org [member]
+  - [ ] have completed the [Inclusive Leadership Training]
+
+- [ ] Once lazy consensus has been achieved, updating the following
+      files in the respective repos:
+  - [ ] [kubernetes/community]: [`sigs.yaml`] and use the [generator doc]
+        to update `README.md` and `OWNERS_ALIASES` files
+  - [ ] [kubernetes/org]: `OWNERS_ALIASES`
```

Comment:
I feel like there's a *lot* of updates required in the org repo for leads, not just the OWNERS_ALIASES. Using myself as a reference, I see that there's also teams.yaml files that need updates in the kubernetes-sigs and kubernetes configs, as well as the milestone-maintainers team under the release team config.

I was adding myself to OWNERS files and teams months after becoming a lead so it might be good to document them all at once :sweat_smile: I think @mrbobbytables mentioned this in his review.

Reply:
> teams.yaml files that need updates in the kubernetes-sigs and kubernetes configs, as well as the milestone-maintainers team under the release team config.

Done :+1: